js/main.js

```diff
@@ -1,70 +1,90 @@
+var player;
+
 window.onload = function() {
+	console.log("setup player");
 	setupPlayer();
 	setupControllerEvents();
 }
 
-
 function setupPlayer() {
-	var conf = {
-		key : "<YOUR_PLAYER_KEY>",
-		source : {
-			// AVC Stream
-			//dash : "https://bitmovin-a.akamaihd.net/content/MI201109210084_1/mpds/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.mpd",
-			// HEVC Stream
-			//dash : "https://bitmovin-a.akamaihd.net/content/multi-codec/hevc/stream.mpd"
-		
-			//Widevine Stream
-			dash: "http://bitmovin-a.akamaihd.net/content/art-of-motion_drm/mpds/11331.mpd",
-			drm: {
-				widevine: {
-					LA_URL: "https://widevine-proxy.appspot.com/proxy",
-				}			
-			}
-		},
-		playback : {
-			autoplay : true
-		},
-		tweaks : {
-			file_protocol : true,
-			app_id : "com.bitmovin.demo.webapp"
-		}
-	};
+	var config = {
+			  key: '7a94a7fd-61a6-40e1-b357-3d19e33ebcb1',
+			  analytics: {
+			    key: '8b4faeb2-0b7a-443e-9614-6fb40fdf5079',
+			    videoId: 'YOUR VIDEO ID',
+			    title: 'A descriptive video title'
+			  },
+			  playback : {
+				autoplay : true
+			  },
+			  tweaks : {
+				file_protocol : true,
+				app_id : "com.bitmovin.demo.webapp"
+			  }
+			};
 
-	window.player = bitmovin.player("player");
-	player.setup(conf).then(function(value) {
-		// Success
-		console.log("Successfully created bitmovin player instance");
-	}, function(reason) {
-		// Error!
-		console.log("Error while creating bitmovin player instance");
-	});
-	
-	player.addEventHandler(bitmovin.player.EVENT.ON_WARNING, function(data){
+			var container = document.getElementById('my-player');
+			player = new bitmovin.player.Player(container, config);
+
+			var source = {
+				// AVC Stream
```

Comment:
remove comments

Reply:
This is from the original repo to show all possible streams, without comments it would be to identify. I would like to keep them if that is okay!